### PR TITLE
Update tool version to preview1

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @muratg \ @prafullbhosale

File: src/Microsoft.Extensions.CodeGeneration.EntityFrameworkCore/project.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
       }
     }
   },
-  "packInclude": {
+  "packOptions": {
     "Templates/": "Templates/"
```
Comment (davidfowl) [inline src/Microsoft.Extensions.CodeGeneration.EntityFrameworkCore/project.json]: This doesn't look right.

Comment (muratg) [inline src/Microsoft.Extensions.CodeGeneration.EntityFrameworkCore/project.json]: What should it be?

Comment (prafullbhosale) [inline src/Microsoft.Extensions.CodeGeneration.EntityFrameworkCore/project.json]: I thought packOptions needs the 
```
"files": {
      "mappings": {
      }
}
```
Also, packOptions will not preserve the directory structure dotnet/cli/issues/2829

Comment (pranavkm) [inline src/Microsoft.Extensions.CodeGeneration.EntityFrameworkCore/project.json]: I'll revert this.

**Comment by pranavkm** [comment]
Updated.

**Comment by prafullbhosale** [comment]
Looks good to me. 
:shipit: 
cc @phenning

**Comment by pranavkm** [comment]
No code change!

**Comment by pranavkm** [comment]
@muratg - updated

**Comment by muratg** [comment]
:shipit:

**Pull request closed without merging**